Lushu/src/main/kotlin/lushu/ContextGrammar/App.kt

```diff
@@ -0,0 +1,6 @@
+package lushu.ContextGrammar
+import lushu.ContextGrammar.Grammar.Grammar
+fun main (args: Array<String>){
+    var grammar = Grammar.fromStdinTrain()
+    grammar = Grammar.fromStdin()
```

Comment:
```suggestion
    val grammar = Grammar.trainFromStdin()
    grammar.consumeStdin()
```

Reply:
Actually, why don't we train from a file instead, and only use stdin to perform the obfuscation simulation?